### PR TITLE
Documentation: project structure

### PR DESCRIPTION
**Pull request by johannesvollmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
-# js-sadness
-These are things in JS which make us sad.
-
-This repository uses `jekyll` which comes with github pages. 
+@@ -5,4 +5,15 @@ This repository uses `jekyll` which comes with github pages.
```
Comment (deflectn) [inline README.md]:
```suggestion
@@ -5,4 +5,15 @@ This repository uses [`jekyll`](https://jekyllrb.com/) which comes with github pages.
```

Comment (deflectn) [reply README.md]: only change of #2

File: README.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
-With jekyll instealled, run `jekyll serve` inside this project directory.
\ No newline at end of file
+With jekyll instealled, run `jekyll serve` inside this project directory, 
+and then visit [localhost on port 4000](http://localhost:4000/).
+
+## Project Structure
+
+### _posts
+This folder contains all sadnesses. Add a file to this folder
+in order to contribute sad content.
```
Comment (deflectn) [inline README.md]:
```suggestion
in order to contribute sadness.
```

**Comment by johannesvollmer** [comment]
still checking??? for over 1 hour??

**Comment by johannesvollmer** [comment]
it's just the README, come on github

**Comment by johannesvollmer** [comment]
It's been two hours now

**Comment by johannesvollmer** [comment]
is this not configured correctly?

**Pull request merged**